src/pages/workspace/distanceRates/PolicyDistanceRatesPage.tsx

```diff
@@ -0,0 +1,280 @@
+                icon: Expensicons.DocumentSlash,
+                onSelected: enableRates,
+            });
+        }
+
+        return options;
+    };
+
+    const headerButtons = (
+        <View style={[styles.w100, styles.flexRow, isSmallScreenWidth && styles.mb3]}>
+            {selectedDistanceRates.length === 0 ? (
+                <>
+                    <Button
+                        medium
+                        text={translate('workspace.distanceRates.addRate')}
+                        onPress={addRate}
+                        style={[styles.mr3, isSmallScreenWidth && styles.flexGrow1]}
+                        icon={Expensicons.Plus}
+                        iconStyles={[styles.mr2]}
+                        success
+                    />
+
+                    <Button
+                        medium
+                        text={translate('workspace.common.settings')}
+                        onPress={openSettings}
+                        style={[isSmallScreenWidth && styles.flexGrow1]}
+                        icon={Expensicons.Gear}
+                        iconStyles={[styles.mr2]}
+                    />
+                </>
+            ) : (
+                <ButtonWithDropdownMenu<WorkspaceDistanceRatesBulkActionType>
+                    shouldAlwaysShowDropdownMenu
+                    pressOnEnter
+                    customText={`${selectedDistanceRates.length} ${translate('workspace.distanceRates.selected')}`}
+                    buttonSize={CONST.DROPDOWN_BUTTON_SIZE.MEDIUM}
+                    onPress={() => null}
+                    options={getBulkActionsButtonOptions()}
+                    buttonRef={dropdownButtonRef}
+                    style={[isSmallScreenWidth && styles.flexGrow1]}
+                    wrapperStyle={styles.w100}
+                />
+            )}
+        </View>
+    );
+
+    return (
+        <AdminPolicyAccessOrNotFoundWrapper policyID={route.params.policyID}>
+            <PaidPolicyAccessOrNotFoundWrapper policyID={route.params.policyID}>
+                <ScreenWrapper
+                    includeSafeAreaPaddingBottom={false}
+                    style={[styles.defaultModalContainer]}
+                    testID={PolicyDistanceRatesPage.displayName}
+                    shouldShowOfflineIndicatorInWideScreen
+                >
+                    <HeaderWithBackButton
+                        icon={Illustrations.CarIce}
+                        title={translate('workspace.common.distanceRates')}
+                        shouldShowBackButton={false}
+                    >
+                        {!isSmallScreenWidth && headerButtons}
+                    </HeaderWithBackButton>
+                    {isSmallScreenWidth && <View style={[styles.ph5]}>{headerButtons}</View>}
+                    <Text style={[styles.pl5, styles.pb2, styles.pt4, styles.textSupporting]}>{translate('workspace.distanceRates.centrallyManage')}</Text>
```

Comment:
Got it, yeah ideally that top header bar has a fixed height of 80px and thus it wouldn't shift any content below it around. cc @Expensify/design for a gut check but I think they would agree!

Reply:
Yup! It looks like something wacky is going on with the header because you can see how the description text for the Distance rates page is a lot lower than for Tags and Categories.

Reply:
@getusha According to designs header has fixed height of 80px but the text below also has 12px of top margin - this isnt the case in tags or categories. Is it an oversight on other screens? Should I add it here or not? Whats the correct version?

<img width="1048" alt="image" src="https://github.com/Expensify/App/assets/23487612/d0cc2e29-9d29-40e1-aead-433e5eed21dd">

<img width="794" alt="image" src="https://github.com/Expensify/App/assets/23487612/b8097b2b-2dc5-4977-9cdf-12d1b19361df">


cc @shawnborton

Reply:
The correct version is that the header area gets a fixed height of 80px, and 12px of space below it.

Reply:
Looks a lot better!